rust-toolchain.toml

```diff
@@ -1,5 +1,5 @@
 [toolchain]
-channel = "1.81"
```

Comment:
What is the motivation for this change? In general, this should match the rust version of the zkVM which is still at 1.81

Reply:
I was just following what used in risc0. Yeah it is odd though that does not match with what supported by the zkvm